### ISSUE TITLE
Crash

### ISSUE THREAD

**Issue by ivysrono** [HenriDellal/emerald-dialer#72]
1.0.10
After Input some keys (related with contacts)
Android 11 Chinese system language.
```
I/ActivityTaskManager( 1501): START u0 {act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] flg=0x10200000 cmp=ru.henridellal.dialer/.DialerActivity} from uid 10206
I/ActivityManager( 1501): Start proc 21176:ru.henridellal.dialer/u0a237 for pre-top-activity {ru.henridellal.dialer/ru.henridellal.dialer.DialerActivity}
D/libMEOW (21176): applied 1 plugins for [ru.henridellal.dialer]:
D/libMEOW (21176): applied 1 plugins for [ru.henridellal.dialer]:
D/KP2AAF  ( 3005): event: 2048, package = ru.henridellal.dialer
I/ActivityTaskManager( 1501): Displayed ru.henridellal.dialer/.DialerActivity: +139ms
D/KP2AAF  ( 3005): event: 2048, package = ru.henridellal.dialer
D/KP2AAF  ( 3005): event: 32, package = ru.henridellal.dialer
D/KP2AAF  ( 3005): URL=androidapp://ru.henridellal.dialer
D/KP2AAF  ( 3005): event: 2048, package = ru.henridellal.dialer
D/KP2AAF  ( 3005): URL=androidapp://ru.henridellal.dialer
D/KP2AAF  ( 3005): event: 2048, package = ru.henridellal.dialer
D/KP2AAF  ( 3005): URL=androidapp://ru.henridellal.dialer
I/chatty  (21176): uid=10237(ru.henridellal.dialer) FinalizerDaemon identical 1 line
I/iorapd  ( 1274): Perfetto TraceBuffer saved to file: /data/misc/iorapd/ru.henridellal.dialer/12/ru.henridellal.dialer.DialerActivity/raw_traces/1628205561766527060.perfetto_trace.pb
D/KP2AAF  ( 3005): event: 2048, package = ru.henridellal.dialer
D/KP2AAF  ( 3005): URL=androidapp://ru.henridellal.dialer
E/AndroidRuntime(21176): Process: ru.henridellal.dialer, PID: 21176
E/AndroidRuntime(21176): 	at ru.henridellal.dialer.ContactsEntryAdapter.getView(ContactsEntryAdapter.java:170)
W/ActivityTaskManager( 1501):   Force finishing activity ru.henridellal.dialer/.DialerActivity
I/ActivityManager( 1501): Process ru.henridellal.dialer (pid 21176) has died: fg  TOP 
I/WindowManager( 1501): WIN DEATH: Window{407b325 u0 ru.henridellal.dialer/ru.henridellal.dialer.DialerActivity}
W/InputDispatcher( 1501): Attempted to unregister already unregistered input channel '407b325 ru.henridellal.dialer/ru.henridellal.dialer.DialerActivity (server)'
W/ActivityTaskManager( 1501): Activity top resumed state loss timeout for ActivityRecord{d4d02b6 u0 ru.henridellal.dialer/.DialerActivity t-1 f}}
D/KP2AAF  ( 3005): event: 2048, package = ru.henridellal.dialer
```

**Comment by HenriDellal** [comment]
Thanks for reporting. Seems like this bug was fixed by me after the release, so the fix is not available for you. I'll create new release tag now to solve this.

**Comment by ivysrono** [comment]
1.0.11 still has this issue.
```
08-06 07:19:48.647 21176 21176 E AndroidRuntime: Process: ru.henridellal.dialer, PID: 21176
08-06 07:19:48.647 21176 21176 E AndroidRuntime: 	at ru.henridellal.dialer.ContactsEntryAdapter.getView(ContactsEntryAdapter.java:170)
08-06 08:03:29.655 21178 21178 E AndroidRuntime: Process: ru.henridellal.dialer, PID: 21178
08-06 08:03:29.655 21178 21178 E AndroidRuntime: 	at ru.henridellal.dialer.ContactsEntryAdapter.getView(ContactsEntryAdapter.java:170)
08-06 08:03:38.610 22078 22078 E AndroidRuntime: Process: ru.henridellal.dialer, PID: 22078
08-06 08:03:38.610 22078 22078 E AndroidRuntime: 	at ru.henridellal.dialer.ContactsEntryAdapter.getView(ContactsEntryAdapter.java:170)
08-06 08:04:36.725 22080 24688 E AndroidRuntime: Process: ru.henridellal.dialer, PID: 22080
08-06 08:04:36.725 22080 24688 E AndroidRuntime: Caused by: java.lang.SecurityException: Permission Denial: opening provider com.android.providers.contacts.ContactsProvider2 from ProcessRecord{61d5197 22080:ru.henridellal.dialer/u0a238} (pid=22080, uid=10238) requires android.permission.READ_CONTACTS or android.permission.WRITE_CONTACTS
08-06 08:04:36.726 22080 24687 E AndroidRuntime: Process: ru.henridellal.dialer, PID: 22080
08-06 08:04:36.726 22080 24687 E AndroidRuntime: Caused by: java.lang.SecurityException: Permission Denial: opening provider com.android.providers.contacts.CallLogProvider from ProcessRecord{61d5197 22080:ru.henridellal.dialer/u0a238} (pid=22080, uid=10238) requires android.permission.READ_CALL_LOG or android.permission.WRITE_CALL_LOG
08-06 08:05:28.738 22081 22081 E AndroidRuntime: Process: ru.henridellal.dialer, PID: 22081
08-06 08:05:28.738 22081 22081 E AndroidRuntime: 	at ru.henridellal.dialer.ContactsEntryAdapter.getView(ContactsEntryAdapter.java:170)
08-06 08:05:33.952 22082 22082 E AndroidRuntime: Process: ru.henridellal.dialer, PID: 22082
08-06 08:05:33.952 22082 22082 E AndroidRuntime: 	at ru.henridellal.dialer.ContactsEntryAdapter.getView(ContactsEntryAdapter.java:170)
08-06 08:16:55.163 24334 24334 E AndroidRuntime: Process: ru.henridellal.dialer, PID: 24334
08-06 08:16:55.163 24334 24334 E AndroidRuntime: 	at ru.henridellal.dialer.ContactsEntryAdapter.getView(ContactsEntryAdapter.java:170)
08-06 12:20:19.828 24839 24839 E AndroidRuntime: Process: ru.henridellal.dialer, PID: 24839
08-06 12:20:19.828 24839 24839 E AndroidRuntime: 	at ru.henridellal.dialer.ContactsEntryAdapter.getView(ContactsEntryAdapter.java:170)
08-06 12:20:24.830 24840 24840 E AndroidRuntime: Process: ru.henridellal.dialer, PID: 24840
08-06 12:20:24.830 24840 24840 E AndroidRuntime: 	at ru.henridellal.dialer.ContactsEntryAdapter.getView(ContactsEntryAdapter.java:170)
08-06 12:24:30.198 24841 24841 E AndroidRuntime: Process: ru.henridellal.dialer, PID: 24841
08-06 12:24:30.198 24841 24841 E AndroidRuntime: 	at ru.henridellal.dialer.ContactsEntryAdapter.getView(ContactsEntryAdapter.java:170)
08-06 12:30:17.702  8071  8071 E AndroidRuntime: Process: ru.henridellal.dialer, PID: 8071
08-06 12:30:17.702  8071  8071 E AndroidRuntime: 	at ru.henridellal.dialer.ContactsEntryAdapter.getView(ContactsEntryAdapter.java:170)
08-06 12:30:32.046  8072  8072 E AndroidRuntime: Process: ru.henridellal.dialer, PID: 8072
08-06 12:30:32.046  8072  8072 E AndroidRuntime: 	at ru.henridellal.dialer.ContactsEntryAdapter.getView(ContactsEntryAdapter.java:170)
08-06 12:31:17.843  8073  8073 E AndroidRuntime: Process: ru.henridellal.dialer, PID: 8073
08-06 12:31:17.843  8073  8073 E AndroidRuntime: 	at ru.henridellal.dialer.ContactsEntryAdapter.getView(ContactsEntryAdapter.java:170)
08-06 12:31:39.229  8074  8074 E AndroidRuntime: Process: ru.henridellal.dialer, PID: 8074
08-06 12:31:39.229  8074  8074 E AndroidRuntime: 	at ru.henridellal.dialer.ContactsEntryAdapter.getView(ContactsEntryAdapter.java:170)
08-06 12:32:07.090 11580 11580 E AndroidRuntime: Process: ru.henridellal.dialer, PID: 11580
08-06 12:32:07.090 11580 11580 E AndroidRuntime: 	at ru.henridellal.dialer.ContactsEntryAdapter.getView(ContactsEntryAdapter.java:170)
08-06 12:32:28.683 11581 11581 E AndroidRuntime: Process: ru.henridellal.dialer, PID: 11581
08-06 12:32:28.683 11581 11581 E AndroidRuntime: 	at ru.henridellal.dialer.ContactsEntryAdapter.getView(ContactsEntryAdapter.java:170)
08-06 12:32:32.957 11582 11582 E AndroidRuntime: Process: ru.henridellal.dialer, PID: 11582
08-06 12:32:32.957 11582 11582 E AndroidRuntime: 	at ru.henridellal.dialer.ContactsEntryAdapter.getView(ContactsEntryAdapter.java:170)
08-06 12:32:45.263 11583 11583 E AndroidRuntime: Process: ru.henridellal.dialer, PID: 11583
08-06 12:32:45.263 11583 11583 E AndroidRuntime: 	at ru.henridellal.dialer.ContactsEntryAdapter.getView(ContactsEntryAdapter.java:170)
08-06 12:33:00.443 11584 11584 E AndroidRuntime: Process: ru.henridellal.dialer, PID: 11584
08-06 12:33:00.443 11584 11584 E AndroidRuntime: 	at ru.henridellal.dialer.ContactsEntryAdapter.getView(ContactsEntryAdapter.java:170)
08-06 12:33:09.105 12631 12631 E AndroidRuntime: Process: ru.henridellal.dialer, PID: 12631
08-06 12:33:09.105 12631 12631 E AndroidRuntime: 	at ru.henridellal.dialer.ContactsEntryAdapter.getView(ContactsEntryAdapter.java:170)
08-10 21:30:26.614 15701 15701 E AndroidRuntime: Process: ru.henridellal.dialer, PID: 15701
08-10 21:30:26.614 15701 15701 E AndroidRuntime: 	at ru.henridellal.dialer.ContactsEntryAdapter.getView(ContactsEntryAdapter.java:172)
08-10 21:32:30.035 25230 28510 E AndroidRuntime: Process: ru.henridellal.dialer, PID: 25230
08-10 21:32:30.035 25230 28510 E AndroidRuntime: Caused by: java.lang.SecurityException: Permission Denial: opening provider com.android.providers.contacts.CallLogProvider from ProcessRecord{11ab067 25230:ru.henridellal.dialer/u0a249} (pid=25230, uid=10249) requires android.permission.READ_CALL_LOG or android.permission.WRITE_CALL_LOG
08-10 21:32:30.035 25230 28511 E AndroidRuntime: Process: ru.henridellal.dialer, PID: 25230
08-10 21:32:30.035 25230 28511 E AndroidRuntime: Caused by: java.lang.SecurityException: Permission Denial: opening provider com.android.providers.contacts.ContactsProvider2 from ProcessRecord{11ab067 25230:ru.henridellal.dialer/u0a249} (pid=25230, uid=10249) requires android.permission.READ_CONTACTS or android.permission.WRITE_CONTACTS
08-10 21:32:41.896 25231 25231 E AndroidRuntime: Process: ru.henridellal.dialer, PID: 25231
08-10 21:32:41.896 25231 25231 E AndroidRuntime: 	at ru.henridellal.dialer.ContactsEntryAdapter.getView(ContactsEntryAdapter.java:172)
08-10 21:37:49.007 25232 25232 E AndroidRuntime: Process: ru.henridellal.dialer, PID: 25232
08-10 21:37:49.007 25232 25232 E AndroidRuntime: 	at ru.henridellal.dialer.ContactsEntryAdapter.getView(ContactsEntryAdapter.java:172)
08-10 21:38:35.987 26949 26949 E AndroidRuntime: Process: ru.henridellal.dialer, PID: 26949
08-10 21:38:35.987 26949 26949 E AndroidRuntime: 	at ru.henridellal.dialer.ContactsEntryAdapter.getView(ContactsEntryAdapter.java:172)
08-10 21:39:30.008 26950 26950 E AndroidRuntime: Process: ru.henridellal.dialer, PID: 26950
08-10 21:39:30.008 26950 26950 E AndroidRuntime: 	at ru.henridellal.dialer.ContactsEntryAdapter.getView(ContactsEntryAdapter.java:172)
08-10 21:39:59.186 26951 26951 E AndroidRuntime: Process: ru.henridellal.dialer, PID: 26951
08-10 21:39:59.186 26951 26951 E AndroidRuntime: 	at ru.henridellal.dialer.ContactsEntryAdapter.getView(ContactsEntryAdapter.java:172)
08-10 21:40:27.081 26952 26952 E AndroidRuntime: Process: ru.henridellal.dialer, PID: 26952
08-10 21:40:27.081 26952 26952 E AndroidRuntime: 	at ru.henridellal.dialer.ContactsEntryAdapter.getView(ContactsEntryAdapter.java:172)
08-10 21:42:22.924 29062 29062 E AndroidRuntime: Process: ru.henridellal.dialer, PID: 29062
08-10 21:42:22.924 29062 29062 E AndroidRuntime: 	at ru.henridellal.dialer.ContactsEntryAdapter.getView(ContactsEntryAdapter.java:172)
08-10 21:43:31.053 29063 29063 E AndroidRuntime: Process: ru.henridellal.dialer, PID: 29063
08-10 21:43:31.053 29063 29063 E AndroidRuntime: 	at ru.henridellal.dialer.ContactsEntryAdapter.getView(ContactsEntryAdapter.java:172)
08-10 21:45:06.774 29064 29064 E AndroidRuntime: Process: ru.henridellal.dialer, PID: 29064
08-10 21:45:06.774 29064 29064 E AndroidRuntime: 	at ru.henridellal.dialer.ContactsEntryAdapter.getView(ContactsEntryAdapter.java:172)
08-10 21:45:38.915 29065 29065 E AndroidRuntime: Process: ru.henridellal.dialer, PID: 29065
08-10 21:45:38.915 29065 29065 E AndroidRuntime: 	at ru.henridellal.dialer.ContactsEntryAdapter.getView(ContactsEntryAdapter.java:172)
08-10 21:46:39.266 29419 29419 E AndroidRuntime: Process: ru.henridellal.dialer, PID: 29419
08-10 21:46:39.266 29419 29419 E AndroidRuntime: 	at ru.henridellal.dialer.ContactsEntryAdapter.getView(ContactsEntryAdapter.java:172)
08-10 21:47:45.567 29420 29420 E AndroidRuntime: Process: ru.henridellal.dialer, PID: 29420
08-10 21:47:45.567 29420 29420 E AndroidRuntime: 	at ru.henridellal.dialer.ContactsEntryAdapter.getView(ContactsEntryAdapter.java:172)
08-10 21:48:08.082 29421 29421 E AndroidRuntime: Process: ru.henridellal.dialer, PID: 29421
08-10 21:48:08.082 29421 29421 E AndroidRuntime: 	at ru.henridellal.dialer.ContactsEntryAdapter.getView(ContactsEntryAdapter.java:172)
08-10 21:48:14.892 29422 29422 E AndroidRuntime: Process: ru.henridellal.dialer, PID: 29422
08-10 21:48:14.892 29422 29422 E AndroidRuntime: 	at ru.henridellal.dialer.ContactsEntryAdapter.getView(ContactsEntryAdapter.java:172)
08-10 21:49:06.348 29423 29423 E AndroidRuntime: Process: ru.henridellal.dialer, PID: 29423
08-10 21:49:06.348 29423 29423 E AndroidRuntime: 	at ru.henridellal.dialer.ContactsEntryAdapter.getView(ContactsEntryAdapter.java:172)
08-10 21:49:14.719 29424 29424 E AndroidRuntime: Process: ru.henridellal.dialer, PID: 29424
08-10 21:49:14.719 29424 29424 E AndroidRuntime: 	at ru.henridellal.dialer.ContactsEntryAdapter.getView(ContactsEntryAdapter.java:172)
08-10 21:49:41.261 29772 29772 E AndroidRuntime: Process: ru.henridellal.dialer, PID: 29772
08-10 21:49:41.261 29772 29772 E AndroidRuntime: 	at ru.henridellal.dialer.ContactsEntryAdapter.getView(ContactsEntryAdapter.java:172)
08-10 21:51:31.136 29773 29773 E AndroidRuntime: Process: ru.henridellal.dialer, PID: 29773
08-10 21:51:31.136 29773 29773 E AndroidRuntime: 	at ru.henridellal.dialer.ContactsEntryAdapter.getView(ContactsEntryAdapter.java:172)
08-10 21:54:40.241 29774 29774 E AndroidRuntime: Process: ru.henridellal.dialer, PID: 29774
08-10 21:54:40.241 29774 29774 E AndroidRuntime: 	at ru.henridellal.dialer.ContactsEntryAdapter.getView(ContactsEntryAdapter.java:172)
08-10 21:56:02.012 29775 29775 E AndroidRuntime: Process: ru.henridellal.dialer, PID: 29775
08-10 21:56:02.012 29775 29775 E AndroidRuntime: 	at ru.henridellal.dialer.ContactsEntryAdapter.getView(ContactsEntryAdapter.java:172)
08-10 21:59:33.086 29776 29776 E AndroidRuntime: Process: ru.henridellal.dialer, PID: 29776
08-10 21:59:33.086 29776 29776 E AndroidRuntime: 	at ru.henridellal.dialer.ContactsEntryAdapter.getView(ContactsEntryAdapter.java:172)
08-10 22:00:38.186 30041 30041 E AndroidRuntime: Process: ru.henridellal.dialer, PID: 30041
08-10 22:00:38.186 30041 30041 E AndroidRuntime: 	at ru.henridellal.dialer.ContactsEntryAdapter.getView(ContactsEntryAdapter.java:172)
```

**Comment by HenriDellal** [comment]
Please could you run adb logcat again, but without filtering by package name? Copy only the information related to dialer's crash.

By the way, I've noticed that you had some permissions disabled, that may cause application malfunctioning as well (and can cause #71 as well). If permissions are currently disabled, please enable permissions and try to check for both bugs again.

**Comment by ivysrono** [comment]
I'm sure that I have enabled all permissions.

`.\adb.exe logcat *:E > 1.log`

```
--------- beginning of main
08-11 20:48:55.177  1835  1999 E WifiVendorHal: getWifiLinkLayerStats_1_3_Internal(l.973) failed {.code = ERROR_NOT_SUPPORTED, .description = }
--------- beginning of system
08-11 20:48:58.200  1835  1999 E WifiVendorHal: getWifiLinkLayerStats_1_3_Internal(l.973) failed {.code = ERROR_NOT_SUPPORTED, .description = }
08-11 20:49:01.224  1835  1999 E WifiVendorHal: getWifiLinkLayerStats_1_3_Internal(l.973) failed {.code = ERROR_NOT_SUPPORTED, .description = }
08-11 20:49:01.360   979  1657 E hwcomposer: [DRMDEV] failed to drmWaitVBlank with dpy_0: -1  
08-11 20:49:02.441   979  1657 E hwcomposer: [DRMDEV] failed to drmWaitVBlank with dpy_0: -1  
08-11 20:49:04.116   979  1657 E hwcomposer: [DRMDEV] failed to drmWaitVBlank with dpy_0: -1  
08-11 20:49:04.248  1835  1999 E WifiVendorHal: getWifiLinkLayerStats_1_3_Internal(l.973) failed {.code = ERROR_NOT_SUPPORTED, .description = }
08-11 20:49:07.272  1835  1999 E WifiVendorHal: getWifiLinkLayerStats_1_3_Internal(l.973) failed {.code = ERROR_NOT_SUPPORTED, .description = }
08-11 20:49:07.412   979  1657 E hwcomposer: [DRMDEV] failed to drmWaitVBlank with dpy_0: -1  
08-11 20:49:10.296  1835  1999 E WifiVendorHal: getWifiLinkLayerStats_1_3_Internal(l.973) failed {.code = ERROR_NOT_SUPPORTED, .description = }
08-11 20:49:13.311  1835  1999 E WifiVendorHal: getWifiLinkLayerStats_1_3_Internal(l.973) failed {.code = ERROR_NOT_SUPPORTED, .description = }
08-11 20:49:13.702 22604 12213 E FirebaseInstanceId: Google Play services missing or without correct permission.
08-11 20:49:14.889  1591  2914 E AudioMTKGainController: setNormalVolume(), invalid param, stream -1, mSceneIndex 0, index 4, devices 2, gainDevice 2, return
08-11 20:49:14.905 22604 12213 E FirebaseInstanceId: Google Play services missing or without correct permission.
08-11 20:49:14.907 17448 17448 E ridellal.diale: Not starting debugger since process cannot load the jdwp agent.
08-11 20:49:14.909  1591  2914 E android.hardware.audio.service.mediatek: misound_crossfade_arsi_query_param_buf_size_by_custom_info(), get param buf size 128 for enh mode SetAudioCustomScene= from file /vendor/etc
08-11 20:49:14.909  1591  2914 E android.hardware.audio.service.mediatek: misound_crossfade_arsi_parsing_param_file_by_custom_info(), parsing file /vendor/etc, custom_info = SetAudioCustomScene=, private buf 0x0
08-11 20:49:14.909  1591  2914 E android.hardware.audio.service.mediatek: product info: key: platform, value: MT6893
08-11 20:49:14.909  1591  2914 E android.hardware.audio.service.mediatek: product info: key: device, value: phhgsi_arm64_ab
08-11 20:49:14.909  1591  2914 E android.hardware.audio.service.mediatek: product info: key: model, value: Phh-Treble vanilla
08-11 20:49:14.909  1591  2914 E aurisys_lib_handler: arsi_parsing_param_file_by_custom_info fail, retval -22
08-11 20:49:14.909  1591  2914 E android.hardware.audio.service.mediatek: misound_arsi_query_param_buf_size_by_custom_info(), get param buf size 128 for enh mode SetAudioCustomScene= from file /vendor/etc
08-11 20:49:14.909  1591  2914 E android.hardware.audio.service.mediatek: misound_arsi_parsing_param_file_by_custom_info(), parsing file /vendor/etc, custom_info = SetAudioCustomScene=, private buf 0xf7366030
08-11 20:49:14.909  1591  2914 E android.hardware.audio.service.mediatek: misoundfx parse param end misound_arsi_parsing_param_file_by_custom_info 207
08-11 20:49:14.909  1591  2914 E android.hardware.audio.service.mediatek: misound_crossfade_arsi_create_handler entry.
08-11 20:49:14.909  1591  2914 E android.hardware.audio.service.mediatek: misound_crossfade_init done.
08-11 20:49:14.909  1591  2914 E android.hardware.audio.service.mediatek: misoundfx crossfade create handler end. misound_crossfade_arsi_create_handler 242
08-11 20:49:14.909  1591  2914 E android.hardware.audio.service.mediatek: misound4_arsi_create_handler entry.
08-11 20:49:14.909  1591  2914 E android.hardware.audio.service.mediatek: misound4_arsi_create_handler stream_type : -1
08-11 20:49:14.910  1591  2914 E android.hardware.audio.service.mediatek: misound4_fx_lib resfile reference 268
08-11 20:49:14.910  1591  2914 E android.hardware.audio.service.mediatek: misound4_fx_lib dsp lib version: 4.0.11
08-11 20:49:14.910  1591  2914 E android.hardware.audio.service.mediatek: misound4_fx_lib dsp date: 2020.9.24
08-11 20:49:14.910  1591  2914 E android.hardware.audio.service.mediatek: misound4_spk_init done.
08-11 20:49:14.910  1591  2914 E android.hardware.audio.service.mediatek: misoundfx create handler end. misound_arsi_create_handler 288
08-11 20:49:14.923 22604 12213 E FirebaseInstanceId: Google Play services missing or without correct permission.
08-11 20:49:14.930  1591  2914 E AudioDspStreamManager: startTasks_l() but size 0
08-11 20:49:14.931  1591  2914 E android.hardware.audio.service.mediatek: misound_arsi_set_key_value_pair(), key value pair = MisoundEnable=1
08-11 20:49:14.931  1591  2914 E android.hardware.audio.service.mediatek: misound_arsi_set_key_value_pair(), key: MisoundEnable
08-11 20:49:14.931  1591  2914 E android.hardware.audio.service.mediatek: misound_arsi_set_key_value_pair(), value: 1
08-11 20:49:14.932  1591  2914 E android.hardware.audio.service.mediatek: misound_arsi_set_key_value_pair(), key value pair = SpkMode=1
08-11 20:49:14.932  1591  2914 E android.hardware.audio.service.mediatek: misound_arsi_set_key_value_pair(), key: SpkMode
08-11 20:49:14.932  1591  2914 E android.hardware.audio.service.mediatek: misound_arsi_set_key_value_pair(), value: 1
08-11 20:49:14.932  1591  2914 E android.hardware.audio.service.mediatek: misound_arsi_set_key_value_pair(), misound select spk mode type: 1
08-11 20:49:14.932  1591  2914 E android.hardware.audio.service.mediatek: misound_arsi_set_key_value_pair(), key value pair = SystemSound=0
08-11 20:49:14.932  1591  2914 E android.hardware.audio.service.mediatek: misound_arsi_set_key_value_pair(), key: SystemSound
08-11 20:49:14.932  1591  2914 E android.hardware.audio.service.mediatek: misound_arsi_set_key_value_pair(), value: 0
08-11 20:49:14.932  1591  2914 E android.hardware.audio.service.mediatek: misound_arsi_set_key_value_pair(), misound select spk set system sound: 0
08-11 20:49:14.932  1591  2914 E android.hardware.audio.service.mediatek: misound_crossfade_arsi_set_key_value_pair(), key value pair = misoundCrossFadeMode=0
08-11 20:49:14.932  1591  2914 E android.hardware.audio.service.mediatek: misound_crossfade_arsi_set_key_value_pair(), key: misoundCrossFadeMode
08-11 20:49:14.932  1591  2914 E android.hardware.audio.service.mediatek: misound_crossfade_arsi_set_key_value_pair(), value: 0
08-11 20:49:14.932 17448 21670 E QT      : [QT]file does not exist
08-11 20:49:14.932 17448 21670 E QT      : [QT]file does not exist
08-11 20:49:16.335  1835  1999 E WifiVendorHal: getWifiLinkLayerStats_1_3_Internal(l.973) failed {.code = ERROR_NOT_SUPPORTED, .description = }
08-11 20:49:16.708   979  1657 E hwcomposer: [DRMDEV] failed to drmWaitVBlank with dpy_0: -1  
--------- beginning of crash
08-11 20:49:18.238 17448 17448 E AndroidRuntime: FATAL EXCEPTION: main
08-11 20:49:18.238 17448 17448 E AndroidRuntime: Process: ru.henridellal.dialer, PID: 17448
08-11 20:49:18.238 17448 17448 E AndroidRuntime: java.lang.IndexOutOfBoundsException: setSpan (11 ... 14) ends beyond length 13
08-11 20:49:18.238 17448 17448 E AndroidRuntime: 	at android.text.SpannableStringInternal.checkRange(SpannableStringInternal.java:491)
08-11 20:49:18.238 17448 17448 E AndroidRuntime: 	at android.text.SpannableStringInternal.setSpan(SpannableStringInternal.java:197)
08-11 20:49:18.238 17448 17448 E AndroidRuntime: 	at android.text.SpannableStringInternal.setSpan(SpannableStringInternal.java:184)
08-11 20:49:18.238 17448 17448 E AndroidRuntime: 	at android.text.SpannableString.setSpan(SpannableString.java:60)
08-11 20:49:18.238 17448 17448 E AndroidRuntime: 	at ru.henridellal.dialer.ContactsEntryAdapter.getView(ContactsEntryAdapter.java:172)
08-11 20:49:18.238 17448 17448 E AndroidRuntime: 	at android.widget.AbsListView.obtainView(AbsListView.java:2389)
08-11 20:49:18.238 17448 17448 E AndroidRuntime: 	at android.widget.ListView.makeAndAddView(ListView.java:2067)
08-11 20:49:18.238 17448 17448 E AndroidRuntime: 	at android.widget.ListView.fillDown(ListView.java:793)
08-11 20:49:18.238 17448 17448 E AndroidRuntime: 	at android.widget.ListView.fillSpecific(ListView.java:1504)
08-11 20:49:18.238 17448 17448 E AndroidRuntime: 	at android.widget.ListView.layoutChildren(ListView.java:1802)
08-11 20:49:18.238 17448 17448 E AndroidRuntime: 	at android.widget.AbsListView.onLayout(AbsListView.java:2186)
08-11 20:49:18.238 17448 17448 E AndroidRuntime: 	at android.view.View.layout(View.java:22844)
08-11 20:49:18.238 17448 17448 E AndroidRuntime: 	at android.view.ViewGroup.layout(ViewGroup.java:6389)
08-11 20:49:18.238 17448 17448 E AndroidRuntime: 	at android.widget.RelativeLayout.onLayout(RelativeLayout.java:1103)
08-11 20:49:18.238 17448 17448 E AndroidRuntime: 	at android.view.View.layout(View.java:22844)
08-11 20:49:18.238 17448 17448 E AndroidRuntime: 	at android.view.ViewGroup.layout(ViewGroup.java:6389)
08-11 20:49:18.238 17448 17448 E AndroidRuntime: 	at android.widget.FrameLayout.layoutChildren(FrameLayout.java:332)
08-11 20:49:18.238 17448 17448 E AndroidRuntime: 	at android.widget.FrameLayout.onLayout(FrameLayout.java:270)
08-11 20:49:18.238 17448 17448 E AndroidRuntime: 	at android.view.View.layout(View.java:22844)
08-11 20:49:18.238 17448 17448 E AndroidRuntime: 	at android.view.ViewGroup.layout(ViewGroup.java:6389)
08-11 20:49:18.238 17448 17448 E AndroidRuntime: 	at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1829)
08-11 20:49:18.238 17448 17448 E AndroidRuntime: 	at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1673)
08-11 20:49:18.238 17448 17448 E AndroidRuntime: 	at android.widget.LinearLayout.onLayout(LinearLayout.java:1582)
08-11 20:49:18.238 17448 17448 E AndroidRuntime: 	at android.view.View.layout(View.java:22844)
08-11 20:49:18.238 17448 17448 E AndroidRuntime: 	at android.view.ViewGroup.layout(ViewGroup.java:6389)
08-11 20:49:18.238 17448 17448 E AndroidRuntime: 	at android.widget.FrameLayout.layoutChildren(FrameLayout.java:332)
08-11 20:49:18.238 17448 17448 E AndroidRuntime: 	at android.widget.FrameLayout.onLayout(FrameLayout.java:270)
08-11 20:49:18.238 17448 17448 E AndroidRuntime: 	at com.android.internal.policy.DecorView.onLayout(DecorView.java:784)
08-11 20:49:18.238 17448 17448 E AndroidRuntime: 	at android.view.View.layout(View.java:22844)
08-11 20:49:18.238 17448 17448 E AndroidRuntime: 	at android.view.ViewGroup.layout(ViewGroup.java:6389)
08-11 20:49:18.238 17448 17448 E AndroidRuntime: 	at android.view.ViewRootImpl.performLayout(ViewRootImpl.java:3475)
08-11 20:49:18.238 17448 17448 E AndroidRuntime: 	at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:2943)
08-11 20:49:18.238 17448 17448 E AndroidRuntime: 	at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1948)
08-11 20:49:18.238 17448 17448 E AndroidRuntime: 	at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:8181)
08-11 20:49:18.238 17448 17448 E AndroidRuntime: 	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:972)
08-11 20:49:18.238 17448 17448 E AndroidRuntime: 	at android.view.Choreographer.doCallbacks(Choreographer.java:796)
08-11 20:49:18.238 17448 17448 E AndroidRuntime: 	at android.view.Choreographer.doFrame(Choreographer.java:731)
08-11 20:49:18.238 17448 17448 E AndroidRuntime: 	at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:957)
08-11 20:49:18.238 17448 17448 E AndroidRuntime: 	at android.os.Handler.handleCallback(Handler.java:938)
08-11 20:49:18.238 17448 17448 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:99)
08-11 20:49:18.238 17448 17448 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:223)
08-11 20:49:18.238 17448 17448 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:7664)
08-11 20:49:18.238 17448 17448 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
08-11 20:49:18.238 17448 17448 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
08-11 20:49:18.238 17448 17448 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
08-11 20:49:19.361  1835  1999 E WifiVendorHal: getWifiLinkLayerStats_1_3_Internal(l.973) failed {.code = ERROR_NOT_SUPPORTED, .description = }
08-11 20:49:19.422   979  1657 E hwcomposer: [DRMDEV] failed to drmWaitVBlank with dpy_0: -1  
08-11 20:49:21.319  1591  3266 E android.hardware.audio.service.mediatek: misound_arsi_set_key_value_pair(), key value pair = MisoundEnable=1
08-11 20:49:21.319  1591  3266 E android.hardware.audio.service.mediatek: misound_arsi_set_key_value_pair(), key: MisoundEnable
08-11 20:49:21.319  1591  3266 E android.hardware.audio.service.mediatek: misound_arsi_set_key_value_pair(), value: 1
08-11 20:49:21.319  1591  3266 E android.hardware.audio.service.mediatek: misound_arsi_set_key_value_pair(), key value pair = EQEnable=1
08-11 20:49:21.319  1591  3266 E android.hardware.audio.service.mediatek: misound_arsi_set_key_value_pair(), key: EQEnable
08-11 20:49:21.319  1591  3266 E android.hardware.audio.service.mediatek: misound_arsi_set_key_value_pair(), value: 1
08-11 20:49:21.320  1591  3266 E android.hardware.audio.service.mediatek: misound_arsi_set_key_value_pair(), key value pair = EQGains=0.0#0.0#0.0#0.0#0.0#0.0#0.0#
08-11 20:49:21.320  1591  3266 E android.hardware.audio.service.mediatek: misound_arsi_set_key_value_pair(), key: EQGains
08-11 20:49:21.320  1591  3266 E android.hardware.audio.service.mediatek: misound_arsi_set_key_value_pair(), value: 0.0#0.0#0.0#0.0#0.0#0.0#0.0#
08-11 20:49:21.438  1591  3266 E android.hardware.audio.service.mediatek: misound4_arsi_destroy_handler misound_crossfade_arsi_destroy_handler
08-11 20:49:21.438  1591  3266 E android.hardware.audio.service.mediatek: misound4_arsi_destroy_handler misound_arsi_destroy_handler
08-11 20:49:22.382  1835  1999 E WifiVendorHal: getWifiLinkLayerStats_1_3_Internal(l.973) failed {.code = ERROR_NOT_SUPPORTED, .description = }
08-11 20:49:22.524   979  1657 E hwcomposer: [DRMDEV] failed to drmWaitVBlank with dpy_0: -1  

```

**Comment by HenriDellal** [comment]
Fixed in 1.0.12

**Comment by ivysrono** [comment]
Not publish to F-Droid yet?
